Automattic/simplenote-android
# Issue/422 switch trash icon

### Fix
Switch empty and restore trash icons as described in https://github.com/Automattic/simplenote-android/issues/422.

### Test
1. Open navigation drawer.
2. Tap ***All Notes*** item.
3. Tap any note in ***All Notes*** list.
4. Tap trash icon in action bar.
5. Open navigation drawer.
6. Tap ***Trash*** item.
7. Long-press trash icon.
8. Notice "Empty trash" message.
9. Tap trashed note in ***Trash*** list.
10. Long-press restore icon.
11. Notice "Restore note" message.
12. Tap restore icon.
